### ISSUE TITLE
Bug following interactive tutorial “Interactive Tutorial - Exposing Your App”

### ISSUE THREAD

**Issue by onslowcodes** [kubernetes/website#39223]
The port is notgetting exposed following the intructions exactly: "curl $(minikube ip):$NODE_PORT

**Comment by tamilselvan1102** [comment]
Describe the issue detailly

**Comment by devarsh10** [comment]
Hey @onslowcodes,
Here, we are getting response "Hello kubernetes bootcamp!" which means our port is exposed. Though if you search minikubeip:NodePort in the browser it may not work.
But, as mentioned here, "we get a response from the server. The Service is exposed."
Meaning, the service is exposed. 

![image](https://user-images.githubusercontent.com/83413047/216775893-1321244e-a18b-46ca-aed5-9ca009f3137b.png)

HOPE THAT HELPS,
Regards.

**Comment by sftim** [comment]
If I wanted to reproduce this @onslowcodes, how would I reproduce this and what would I see to know that it has not worked as expected?

/triage needs-information
/language en

**Comment by sftim** [comment]
/retitle Bug following interactive tutorial “Interactive Tutorial - Exposing Your App”

**Comment by tengqm** [comment]
/close

Won't fix because we're [removing Katacoda](https://blog.k8s.io/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/).

**Comment by jaxtonw** [comment]
This is related to Docker not exposing ports properly. If minikube is installed with Docker, you will likely run into this issue. One could manually 

Here's the easiest fix: https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel
TL;DR: Run `minikube service kubernetes-bootcamp --url` in another terminal window, and then curl with the url the `minikube service` returns instead of the suggested one in the tutorial. 

This issue isn't exclusive to the Katacoda environment, and I was experiencing this with WSL2 + Docker Desktop. I'd suggest that the docs add a little disclaimer saying something like "Are you failing to connect to the service? If you installed minikube with Docker, see here: ..."
